js_modules/dagster-ui/packages/ui-core/src/asset-graph/CollapsedGroupNode.tsx

```diff
@@ -84,7 +84,7 @@ export const CollapsedGroupNode = ({
     <ContextMenuWrapper menu={menu} stopPropagation>
       <CollapsedGroupNodeContainer
         onClick={(e) => {
-          onExpand();
+          onExpand(e);
```

Comment:
I think Instead of piggy backing off of `onExpand` we should add a new prop since this new behavior isn't actually expanding. Then we can keep the `e.meta` check here.


```
if (e.metaKey) {
  toggleSelectAllNodes();
} else {
  onExpand();
}
```